### PR TITLE
Kolja/update events

### PR DESCRIPTION
**Pull request by kolja-lucht**
Added the edit and update methods in a new branch

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -13,6 +13,17 @@ def create
...
 	end
 
+	def edit
+    @event = Event.find(params[:id])
+  end
+
+  def update
+    @event = Event.find(params[:id])
+    @event.update(event_params)
+
```
Comment (julesinberlin) [inline app/controllers/events_controller.rb]: I would suggest to delete the extra blank line 23

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+    redirect_to event_path(@event)
+  end
+
+
+	def destroy
+		@event = Event.find(params[:id])
+		@event.destroy
+
```
Comment (julesinberlin) [inline app/controllers/events_controller.rb]: I would get rid of the extra blank line 31 as well

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (kolja-lucht) [reply app/controllers/events_controller.rb]: Fixed with rubocop e74418477e56ceb387e855a1c9b90138b9f90713

File: app/controllers/events_controller.rb (hunk 2 above)
Comment (kolja-lucht) [reply app/controllers/events_controller.rb]: Fixed with rubocop e74418477e56ceb387e855a1c9b90138b9f90713

**Pull request merged**